src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1211,11 +1213,7 @@ export class ResultsViewPageStore {
         ],
         invoke: () => {
             // we get mutations with mutations endpoint, all other alterations with this one, so filter out mutation genetic profile
```

Comment:
Thanks @kalletlak for this quick fixing! Looks good to me!

Maybe can edit this comment to make it more clear like: 
"we get mutations with mutations endpoint, structural variants and fusions with structural variant endpoint, all other alterations with this one, so filter out mutation genetic profile and structural variant profiles "